src/components/avatar.tsx

```diff
@@ -62,3 +62,49 @@ export const Avatar = (props: AvatarProps) => {
     </MjmlRaw>
   );
 };
+
+export const SmallAvatar = (props: AvatarProps) => {
```

Comment:
I think this one can combine with Avatar? What is the effect of unwrapping `<MjmlRaw />`